src/components/CsvBlockTable.tsx

```diff
@@ -0,0 +1,357 @@
+            service.
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>is_satellite_provider</Td>
+          <Td>boolean</Td>
+          <Td>
+            <Strong>Deprecated.</Strong>
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>postal_code</Td>
+          <Td>string</Td>
+          <Td>
+            A postal code close to the user&apos;s location. We return the
+            first 3 characters for Canadian postal codes. We return the first
+            2-4 characters (outward code) for postal codes in the United
+            Kingdom.
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                isDisabled
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>latitude*</Td>
+          <Td>decimal</Td>
+          <Td>
+            The approximate
+            {' '}
+            <A
+              href="https://en.wikipedia.org/wiki/World_Geodetic_System"
+              rel="nofollow noopener noreferrer"
+              target="_blank"
+            >
+              WGS84
+            </A>
+            {' '}
+            latitude of the location associated with the network.
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                isDisabled
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>longitude*</Td>
+          <Td>decimal</Td>
+          <Td>
+            The approximate
+            {' '}
+            <A
+              href="https://en.wikipedia.org/wiki/World_Geodetic_System"
+              rel="nofollow noopener noreferrer"
+              target="_blank"
+            >
+              WGS84
+            </A>
+            {' '}
+            longitude of the location associated with the network.
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                isDisabled
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>accuracy_radius</Td>
+          <Td>integer</Td>
+          <Td>
+            The radius in kilometers around the specified location where the IP
```

Comment:
The original text says: 
The approximate accuracy radius, in kilometers, around the latitude and longitude for the geographical entity (country, subdivision, city or postal code) associated with the IP address. We have a 67% confidence that the location of the end-user falls within the area defined by the accuracy radius and the latitude and longitude coordinates.

Not sure if this was changed on purpose. If so you can ignore this

Reply:
same issue as the comment above. On the https://dev.maxmind.com/geoip/geoip2/geoip2-city-country-csv-databases/ page it shows the text I added above for `accuracy_radius`, but on the https://dev.maxmind.com/geoip/geoip2/geoip2-enterprise-csv-database/ page it shows the text you had originally.

Reply:
maybe they are supposed to be different?

Reply:
i'll make a note of this and we'll standardize it during UAT or after the site goes live